src/components/common/HistoryPopup.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from "react";
+import "./styles.css";
+import ReactModal from "react-modal-resizable-draggable";
+
+// react-modal-resizable-draggable   0.1.6  is the dependency used here to make the pop up window funtion properly *SEE BELOW FOR NEEDED CSS
+
+class Popup extends Component {
```

Comment:
Could you do a functional component instead? There are no other class components in the app